### PR TITLE
Alertmanager Configuration: Routing Labels Editor display fix

### PR DESCRIPTION
**Pull request by dtaylor113**
### Regular Expression checkbox should be on same line as row:
image

### Fixed
image

### CODE REVIEW

**Comment by kyoto** [comment]
This was changed by https://github.com/openshift/console/pull/3244. We should make sure that we have the behavior we want both at narrow and wide screen widths.

FYI @sg00dwin

**Comment by kyoto** [comment]
The table header widths also need to be changed so that "VALUE" aligns with the value inputs.

**Comment by dtaylor113** [comment]
> This was changed by #3244. We should make sure that we have the behavior we want both at narrow and wide screen widths.

Thanks @kyoto, I was wondering what happened, I thought I had just missed that the 'regEx` checkbox was displaying on the line below -didn't realize it had changed via #3244 

@sg00dwin, @rhamilto, & @cshinn are we happy with the 'regular expression' checkbox being displayed on the next line, even at mid/large screens?  I think it looks a little odd, especially with multiple rows:
image

**Comment by sg00dwin** [comment]
> > This was changed by #3244. We should make sure that we have the behavior we want both at narrow and wide screen widths.
> 
> Thanks @kyoto, I was wondering what happened, I thought I had just missed that the 'regEx` checkbox was displaying on the line below -didn't realize it had changed via #3244
> 
> @sg00dwin, @rhamilto, & @cshinn are we happy with the 'regular expression' checkbox being displayed on the next line, even at mid/large screens? I think it looks a little odd, especially with multiple rows:


PR  https://github.com/openshift/console/pull/3244 was to prevent horizontal scroll breakage  and keep the `delete` icon on the same row as the text inputs, at narrow widths. And IMO, moving the `checkbox` "Regular Expression" option [beneath their associated text inputs](https://user-images.githubusercontent.com/1874151/68233672-a4d03d80-ffcd-11e9-8471-c65f3c9c72ba.png) still implies their association.  

But, that said, I don't have a opposition with them all being horizontally positioned together, as long as it doesn't break at mobile widths.

Also, if we do switch this layout we should be consistent and also update the [identical options on the Create Silence form.](https://github.com/openshift/console/blob/master/frontend/public/components/monitoring.tsx#L1224-L1295)

**Comment by dtaylor113** [comment]
> And IMO, moving the `checkbox` "Regular Expression" option [beneath their associated text inputs](https://user-images.githubusercontent.com/1874151/68233672-a4d03d80-ffcd-11e9-8471-c65f3c9c72ba.png) still implies their association.

I'm ok with this as well except there seems to be a lot of padding between rows, I'd prefer to see the rows closer together.

File: frontend/public/components/monitoring/alert-manager-receiver-forms.tsx [hunk 1]
```diff
@@ -173,25 +173,21 @@ const RoutingLabels = ({ routeLabels, setRouteLabels, showAddLabel }) => {
...
-                            onChange={(e) => onRoutingLabelRegexChange(e, _.toNumber(i))}
-                            checked={routeLabel.isRegex}
-                          />
-                          &nbsp; Regular Expression
-                        </label>
-                      </div>
+                {!isDefaultReceiverRouteLabel && (
+                  <div className="col-xs-4">
+                    <div className="form-group co-alert-manager-routing-labels-useregex">
+                      <label className="co-no-bold">
```
Comment (spadgett) [inline frontend/public/components/monitoring/alert-manager-receiver-forms.tsx]: Not new, but you shouldn't need this style if you wrap this in a `<div className="checkbox">`.

File: frontend/public/components/monitoring/_monitoring.scss [hunk 2]
```diff
@@ -229,6 +229,10 @@ $tooltip-background-color: #151515;
   z-index: 1;
 }
 
+.co-alert-manager-routing-labels-useregex {
```
Comment (spadgett) [inline frontend/public/components/monitoring/_monitoring.scss]: This is not following BEM.

Declarations in this file should be alphabetized.

**Comment by dtaylor113** [comment]
Closing this PR as it is displaying what was intended for #3244

**Comment by kyoto** [comment]
FYI @cshinn, @sichvoge

**Pull request closed without merging**